### PR TITLE
Fixed tests

### PR DESCRIPTION
**Pull request by bradbaris**
I think one of the tests were failing because I forgot to make the binaries executable.

### CODE REVIEW

File: test/test.js [hunk 1]
```diff
@@ -23,8 +23,11 @@ test.cb('rebuild the guetzli binaries', t => {
...
 });
 
-test('return path to binary and verify that it is working', async t => {
-	t.true(await binCheck(guetzli, ['--version']));
+test.cb('return path to binary and verify that it is working', t => {
+	binCheck(guetzli, ['--version']).then(result => {
+		t.true(result);
+	});
+	t.end();
 });
```
Comment (1000ch) [inline test/test.js]: Is this change needed?

Comment (bradbaris) [inline test/test.js]: For some reason, the async binCheck isn't being completed before AVA asserts the test. It evaluates to an unresolved Promise, thus `false`. The change to sync lets it actually complete. There are definitely other ways to solve this though.

**Comment by 1000ch** [comment]
@bradbaris 🙌

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "guetzli",
-  "version": "0.1.0",
+  "version": "1.0.0",
```
Comment (sindresorhus) [inline package.json]: Version bumps should be done in a separate commit. I would recommend using https://github.com/sindresorhus/np to make releases safely.

File: test/test.js (hunk 1 above)
Comment (sindresorhus) [inline test/test.js]: @bradbaris This is just hiding the problem. Now it finishes before `t.true()` is even run.

Comment (bradbaris) [inline test/test.js]: Woops, you're right. I thought I double-checked that. Addressed in #7, hopefully.
